examples/commonjs/index.js

```diff
@@ -0,0 +1,15 @@
+const { loadPyodide } = require("pyodide");
```

Comment:
Since the examples are used for tests, this should import everything. I.e. by also importing `version`, hopefully it would reveal the same error as in https://github.com/pyodide/pyodide-webpack-plugin/issues/25